aspnetcore/fundamentals/host/platform-specific-configuration.md

```diff
@@ -161,197 +161,85 @@ Options for hosting startup activation are:
-```
-
-# [macOS](#tab/macos)
-
-```
-/usr/local/share/dotnet/additionalDeps/<ENHANCEMENT_ASSEMBLY_NAME>/shared/Microsoft.NETCore.App/<SHARED_FRAMEWORK_VERSION>/
-```
-
-# [Linux](#tab/linux)
-
-```
-/usr/local/share/dotnet/additionalDeps/<ENHANCEMENT_ASSEMBLY_NAME>/shared/Microsoft.NETCore.App/<SHARED_FRAMEWORK_VERSION>/
-```
-
----
-
-The shared framework version reflects the version of the shared runtime that the target app uses. The shared runtime is shown in the *\*.runtimeconfig.json* file. In the sample app (*HostingStartupApp*), the shared runtime is specified in the *HostingStartupApp.runtimeconfig.json* file.
-
-**List the hosting startup's dependencies file**
-
-The location of the implementation's *\*.deps.json* file is listed in the `DOTNET_ADDITIONAL_DEPS` environment variable.
-
-If the file is placed in the user profile's *.dotnet* folder, set the environment variable's value to:
-
-# [Windows](#tab/windows)
-
-```
-%USERPROFILE%\.dotnet\x64\additionalDeps\
-```
-
-# [macOS](#tab/macos)
-
-```
-/Users/<USER>/.dotnet/x64/additionalDeps/
-```
-
-# [Linux](#tab/linux)
-
-```
-/Users/<USER>/.dotnet/x64/additionalDeps/
-```
-
----
-
-If the file is placed in the .NET Core installation for global use, provide the full path to the file:
-
-# [Windows](#tab/windows)
-
-```
-%PROGRAMFILES%\dotnet\additionalDeps\<ENHANCEMENT_ASSEMBLY_NAME>\shared\Microsoft.NETCore.App\<SHARED_FRAMEWORK_VERSION>\<ENHANCEMENT_ASSEMBLY_NAME>.deps.json
-```
-
-# [macOS](#tab/macos)
-
-```
-/usr/local/share/dotnet/additionalDeps/<ENHANCEMENT_ASSEMBLY_NAME>/shared/Microsoft.NETCore.App/<SHARED_FRAMEWORK_VERSION>/<ENHANCEMENT_ASSEMBLY_NAME>.deps.json
-```
-
-# [Linux](#tab/linux)
-
-```
-/usr/local/share/dotnet/additionalDeps/<ENHANCEMENT_ASSEMBLY_NAME>/shared/Microsoft.NETCore.App/<SHARED_FRAMEWORK_VERSION>/<ENHANCEMENT_ASSEMBLY_NAME>.deps.json
-```
-
----
-
-For the sample app (*HostingStartupApp*) to find the dependencies file (*HostingStartupApp.runtimeconfig.json*), the dependencies file is placed in the user's profile.
-
-# [Windows](#tab/windows)
-
-Set the `DOTNET_ADDITIONAL_DEPS` environment variable to the following value:
-
-```
-%UserProfile%\.dotnet\x64\additionalDeps\StartupDiagnostics\
-```
-
-# [macOS](#tab/macos)
-
-Set the `DOTNET_ADDITIONAL_DEPS` environment variable to the following value:
-
-```
-/Users/<USER>/.dotnet/x64/additionalDeps/StartupDiagnostics/
-```
-
-# [Linux](#tab/linux)
-
-Set the `DOTNET_ADDITIONAL_DEPS` environment variable to the following value:
-
-```
-/Users/<USER>/.dotnet/x64/additionalDeps/StartupDiagnostics/
-```
-
----
+To be able to activate the enhancement without having package reference to it one should use `additionalDeps` feature of the runtime. It allows to extend applications library graph by providing set of additional `.deps.json` files that would get merged with apps own `.deps.json` on startup and would make hosting startup assembly discoverable and loadable.
```

Comment:
```suggestion
To activate the enhancement without having a package reference to the enhancement, use the `additionalDeps` feature of the runtime. `additionalDeps` allows you to:

* Extend the app's library graph by providing a set of additional *\*.deps.json* files to merge with the app's own *\*.deps.json* file on startup.
* Make the hosting startup assembly discoverable and loadable.
```

Note that I try to say consistently "compile-time reference." IIRC, I did that because it applies to both packages and non-packages (class libs). Can "having a package reference" say "having a compile-time reference," or is it better here the way that you have it, even considering my concern about non-package references?